shared-libs/outbound/src/outbound.js

```diff
@@ -198,34 +249,29 @@ module.exports = theLogger => {
      * @param      {<string>}  configName  key used for this config in our app-settings (for logging)
      * @param      {<object>}  record      the couchdb record (ie report) to use
      * @param      {<object>}  recordInfo  the couchdb record's info doc
-     * @return     {<promise>} a promise which is successful if the record was converted and send
-     *                         successfully. Can definitely fail.
+     * @return     {<promise>} a promise which can:
+     *      - resolve with true: outbound was successful and sent
+     *      - resolve with false: outbound didn't error, but wasn't sent out (because it's a duplicate of prior send)
+     *      - reject with error: something went wrong
      */
     send: (config, configName, record, recordInfo) => {
       return Promise.resolve()
         .then(() => mapDocumentToPayload(record, config, configName))
-        .then(payload => sendPayload(payload, config))
-        .then(() => updateInfo(recordInfo, configName))
-        .then(() => logger.info(`Pushed ${record._id} to ${configName}`))
+        .then(payload => {
+          if (alreadySent(payload, configName, recordInfo)) {
+            logger.info(`Not pushing ${record._id} to ${configName} as payload is identical to previous push`);
+            return false;
+          } else {
```

Comment:
No `else` required here, I think.

Reply:
Sorry, sometimes I forget Medic style, I'll change it!